app/models/vm_or_template/operations/relocation.rb

```diff
@@ -47,6 +47,13 @@ def raw_migrate(host, pool = nil, priority = "defaultPriority", state = nil)
       end
     end
 
+    current_host_id = Host.where(:id => host_id).pluck(:uid_ems).first
+    destination_host_id = host.uid_ems
+
+    if current_host_id == destination_host_id
```

Comment:
Just a question: do we need to use `uid_ems`?

If we do just `if host_id == host.id # current host equals destination host` we could avoid extra select from db.